### ISSUE TITLE
Remove reset buttons in mobile view

### ISSUE THREAD

**Issue by a20sarle** [HGustavs/LenaSYS#12285]
The layouts of files can't be changed in mobile view so a suggestion would be to remove the "reset buttons".

![Screenshot_20220509-121608_Samsung Internet](https://user-images.githubusercontent.com/81631818/167390059-c81818ee-36b6-4811-bbf6-1bba8496cbd5.jpg)

**Comment by a20joabo** [comment]
**This issue can not be solved until  #11953 has been merged into the active branch.**

The functionality of the current reset button will be changed after #11953 has been merged into the main branch. 
Make sure that the button being removed in mobile view are the one for resetting the boxes and **not** the one for zooming.
